app/models/unit.rb

```diff
@@ -1617,17 +1599,18 @@ def tasks_as_hash(data)
   # Return all tasks from the database for this unit and given user
   #
   def get_all_tasks_for(user)
+    # TODO (stream) Do we need tutorial here?
```

Comment:
@macite Any suggestions?

Reply:
Here we need the relevant tutorial id through the associated stream